### PR TITLE
[StructuralMechanics] clean python imports

### PR DESCRIPTION
**Pull request by philbucher**
cleaning towards python-only modules

wanted to do this for a while already but didn't have time

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]

File: applications/StructuralMechanicsApplication/python_scripts/simplified_nodal_contact_process.py [hunk 1]
```diff
@@ -3,14 +3,8 @@
...
 
-# Import applications
-import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
-
-# Other imports
-import math
-
 def Factory(settings, Model):
-    if(type(settings) != KratosMultiphysics.Parameters):
+    if not isinstance(settings, KM.Parameters):
```
Comment (armingeiser) [inline applications/StructuralMechanicsApplication/python_scripts/simplified_nodal_contact_process.py]: KM is not defined above?

**Review by armingeiser — COMMENTED** [review]
Looks good. Why did you remove the CreateSolver function in one but not all files?

**Comment by MFusseder** [comment]
Looks fine for me.

File: applications/StructuralMechanicsApplication/python_scripts/simplified_nodal_contact_process.py (hunk 1 above)
Comment (philbucher) [reply applications/StructuralMechanicsApplication/python_scripts/simplified_nodal_contact_process.py]: good catch, thx

File: applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py [hunk 2]
```diff
@@ -7,12 +7,7 @@
 import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
 
 # Importing the base class
-from python_solver import PythonSolver
-
-
-def CreateSolver(model, custom_settings):
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py:13]: @armingeiser I am only removing from the base class
It is not needed here / doesn't make sense here

**Comment by philbucher** [comment]
can someone reapprove pls?

**Review by loumalouomega — APPROVED** [review]

**Review by armingeiser — APPROVED** [review]

**Pull request merged**